js/siteorigin-panels/view/live-editor.js

```diff
@@ -19,7 +20,7 @@ module.exports = Backbone.View.extend( {
      * Render the live editor
      */
     render: function(){
-        this.$el.html( this.template() );
+        this.setElement( this.template() );
```

Comment:
Is this to prevent Backbone from wrapping the template in a `div` element? Should it happen every time view's `render` method is called? If not then maybe it should go in the `initialize` method so that it's only called once for the view.